src/components/GameOver.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from "react";
+import Modal from "react-modal";
+import styled from "styled-components";
+import { useSelector, useDispatch } from "react-redux";
+import { State, store } from "../../public/store";
+
+interface ModalProps {
+  isOpen: boolean;
+}
+const GameOverModal = styled(Modal)<ModalProps>``;
+const Heading = styled.h2``;
+const Winner = styled.h3``;
+const StyledButton = styled.button``;
+
+interface GameOverProps {
+  ws: WebSocket;
+}
+
+const GameOver: React.FC<GameOverProps> = ({ ws }) => {
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const winner = useSelector((state: State) => state.winner);
+  const dispatch = useDispatch();
+  const setModalIsOpenToTrue = () => {
+    setModalIsOpen(true);
+  };
+  const setModalIsOpenToFalse = () => {
+    setModalIsOpen(false);
+  };
+
+  GameOverModal.setAppElement("body");
+
+  useEffect(() => {
+    if (winner) {
+      setModalIsOpenToTrue();
```

Comment:
```suggestion
      setModalIsOpen(true);
```
No need for the helper function